Modules/mmapmodule.c

```diff
@@ -180,6 +183,7 @@ mmap_close_method(mmap_object *self, PyObject *unused)
         self->data = NULL;
```

Comment:
I don't think it's safe to not be holding the GIL here. Someone could be trying to read or write the mmap object while calling `close`.